relay-dynamic-config/src/metrics.rs

```diff
@@ -170,13 +171,113 @@ impl TransactionMetricsConfig {
     }
 }
 
+/// Combined view of global and project-specific metrics extraction configs.
+#[derive(Debug, Clone)]
```

Comment:
```suggestion
#[derive(Debug, Clone, Copy)]
```

Reply:
Depends whether we consider it a reference type or a value type. I don't have a strong opinion, assuming the compiler will optimize both versions to the same thing.